transcom/mymove
# INT Closeout Office For Space Force Bug Fixes

## [B-18497](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-18497)
## [B-18630](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-18630)

## Summary

This PR addresses two issues:
1. Customer's were unable to choose a closeout office for their PPM shipment if they chose the "Space Force" branch of service.
2. Service Counselor's were unable to view/edit a customer's closeout office during counseling if the customer had chosen the "Space Force" branch of service.

Both of these should now be possible.

### How to test

1. Login as a user, create a move with a PPM shipment.
2. On the first screen after choosing "PPM", where you input your estimated weights and departure dates, you should now see a "Closeout Office" field. You can choose whichever you like.
3. Complete the move then login as a Service Counselor with a GBLOC corresponding to your move's origin. Find and select the move.
4. On the shipment card, you should be able to see the customer's chosen closeout office.
5. If you click Edit Shipment, you should be able to edit the closeout office. Edit this to whatever you like, for this example, I'll edit the office to "PPPO NAVSUP FLCN Annapolis - USN" which has a GBLOC of "BGNC" (different from my move's original GBLOC).
6. Complete counseling of the move, then login as user and complete the "Upload PPM Documents" step to send the move to the closeout queue.
7. Now login as a Service Counselor under the GBLOC of the edited closeout office (mine is "BGNC"), change to the "PPM Closeout" tab, and you should now see your move. You should not see this move under any other GBLOCs.

### Edited files:
1. `src/pages/MyMove/PPM/Booking/DateAndLocation/DateAndLocation.jsx`
2. `src/components/Customer/PPM/Booking/DateAndLocationForm/DateAndLocationForm.jsx`
3. `src/components/Office/ShipmentForm/ShipmentForm.jsx`

All others are changes being pulled in from latest MAIN branch.
